test/test_meta.py

```diff
@@ -217,7 +217,6 @@ def test_assert(cond, msg):
         test_assert(isinstance(meta_r, torch.Tensor), f"but real {i}th result is Tensor")
         test_assert(meta_r.dtype == r.dtype, f"but real dtype was {r.dtype}")
         test_assert(meta_r.shape == r.shape, f"but real shape was {r.shape}")
-        # NOTE: stride checking is currently disabled
```

Comment:
delete also the line below?

Reply:
The issue is still relevant as we aren't checking strides still in many cases